src/main/java/com/igot/cb/contentpartner/repository/ContentPartnerRegistrationRepository.java

```diff
@@ -0,0 +1,17 @@
+package com.igot.cb.contentpartner.repository;
+
+import com.igot.cb.contentpartner.entity.ContentPartnerRegistrationEntity;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+
+import java.util.Optional;
+
+public interface ContentPartnerRegistrationRepository extends JpaRepository<ContentPartnerRegistrationEntity, String> {
+    @Query(value = "SELECT * FROM content_partner_registration WHERE data->>'contentPartnerName' = :contentPartnerName", nativeQuery = true)
+    Optional<ContentPartnerRegistrationEntity> findByContentPartnerOrganizationName(@Param("contentPartnerName") String contentPartnerName);
+    @Query(value = "SELECT * FROM content_partner_registration WHERE data->>'email' = :email", nativeQuery = true)
+    Optional<ContentPartnerRegistrationEntity> findByContentPartnerEmail(@Param("email") String email);
```

Comment:
This is fine by code.
But I do not see the possibilities on using this in the UI.
To be successful - you need to give exact value of name or email -- which most of the case may not be done from UI.
We need search API for ES - so that we can search with few chars given in the query for name or email against content partners